link-grammar/parse/prune.c

```diff
@@ -133,19 +133,19 @@ struct prune_context_s
 *
 * The kinds of constraints it checks for are the following:
 *
-*  1) successive connectors on the same disjunct have to go to
+*  1) Successive connectors on the same disjunct have to go to
 *     nearer and nearer words.
 *
-*  2) two deep connectors cannot attach to each other
-*     (A connectors is deep if it is not the first in its list; it
-*     is shallow if it is the first in its list; it is deepest if it
-*     is the last on its list.)
+*  2) Two deep connectors cannot attach to each other; i.e. a deep
+*     connector can only attache to a shallow one, and a shallow
+*     connector can attache to any connector.
```

Comment:
The paranthetical comment should stay.  It's important to know that shallowest==first, deepest==last. in the list.

Reply:
My mistake. Forced-push.